qase-api/pom.xml

```diff
@@ -83,10 +85,9 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.projectlombok</groupId>
-            <artifactId>lombok</artifactId>
-            <version>1.18.24</version>
-            <scope>provided</scope>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
```

Comment:
I have a compilation error when I  use qase reporter in extrernal project without included guice dependency:

>  Could not initialize class io.qase.configuration.QaseModule

Move this dependency to the parent pom, It should fix the error.

Reply:
@savkk , the dependency was moved to the parent pom